app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    new_list = [Person(human["name"], human["age"]) for human in people]
+
+    for person in people:
+        person_name = Person.people[person["name"]]
+        if person.get("wife"):
+            person_name.wife =\
+                Person.people[person["wife"]]
+        if person.get("husband"):
+            person_name.husband =\
+                Person.people[person["husband"]]
```

Comment:
u still using line breaks, which i recommend to avoid. Can you use, maybe, additional variables or some other approaches to get rid of those line breaks ?